README.md

```diff
@@ -77,16 +77,13 @@ Updated 0 as deleted
 # prepare to wait a long time
 ```
 
-Packaging
+Packaging and release
 ---------
 
 For maintainers:
 
-1. Update the `__version__` in setup.py.
-2. Build, check and upload to PyPI as follows:
+1. Update the `__version__` in setup.py. Commit. Push.
+2. Create a release on GitHub for this version.
+3. When published, a [GitHub action](https://github.com/matrix-org/synapse-s3-storage-provider/runs/4312656424?check_suite_focus=true) will build the package and upload to [PyPI](https://pypi.org/project/synapse-s3-storage-provider/).
```

Comment:
```suggestion
3. When published, a [GitHub action](https://github.com/matrix-org/synapse-s3-storage-provider/actions) will build the package and upload to [PyPI](https://pypi.org/project/synapse-s3-storage-provider/).
```

I think it'd be more useful to tell the maintainer where to look for said action, but ymmv

Reply:
Argh, this was meant to be the workflow itself, not a specific run of it. Thanks.